helix-query.yaml

```diff
@@ -41,3 +41,28 @@ indices:
       cq-tags:
         select: head > meta[name="cq-tags"]
         value: attribute(el, "content")
+  search:
```

Comment:
@mayur-satav, considering potential future use cases, could we change this to the **default**? It might streamline things down the road.

Reply:
ok